src/aggregator/client/queue.go

```diff
@@ -133,15 +133,17 @@ func newInstanceQueue(instance placement.Instance, opts Options) instanceQueue {
 		connInstrumentOpts = instrumentOpts.SetMetricsScope(scope.SubScope("connection"))
 		connOpts           = opts.ConnectionOptions().SetInstrumentOptions(connInstrumentOpts)
 		conn               = newConnection(instance.Endpoint(), connOpts)
-		iOpts              = opts.InstrumentOptions()
+		instanceQueueTags  = map[string]string{"instance": instance.ID()}
```

Comment:
Is `instance.ID()` a small bounded set? Wary of cardinality explosion here

Reply:
Yeah I can pull this off.